2.0/s2i/bin/run

```diff
@@ -10,6 +10,16 @@ set -e
 export RACK_ENV=${RACK_ENV:-"production"}
 
 if is_puma_installed; then
+  export_str=$(cgroup-limits.py)
```

Comment:
no, it is not, if cgroup-limits.py fails (and throws a traceback) the export command would still pass and run with the garbage output. Tested.